### PR TITLE
docs(release-highlights): Adds beta release notes.

### LINKED ISSUE #4137: Add beta release highlights
Update our release highlights on org with details about the beta release

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #4137

### CODE REVIEW

**Comment by edonehoo** [comment]
do we want to give more details here or less here than I have so far?

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 1]
```diff
@@ -7,23 +7,51 @@ section: get-started
...
-- Charts 
-
-### Addressed issues throughout the system
-We cleaned up and fixed bugs in a number of areas:
-- We removed Sass global variables in favor of our tokens.
-- We renamed variables that previously had top/right/bottom/left in the name (for example, `PaddingTop`) to instead match inline/block start/end properties (for example, `PaddingBlockStart`). This change is a result of our introduction of [RTL support.](/developer-resources/right-to-left-handbook)
-- We removed the deprecated Sass `@import` rule in favor of new recommendations, `@use` and `@forward`.
-- We fixed various bugs that were reported following the initial alpha release.
+
+We extended PatternFly 6 styles to Topology. Now, all PatternFly elements are covered for the release, and all related design resources can be found in [our PatternFly 6 Figma kit.]()
```
Comment (mcoker) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:17]: This suggestion could be way off base and/or unnecessary, but if this is just referring to the patternfly stuff in topology, I wonder if there is another word to use other than "elements" to disambiguate from [PFE/patternfly elements the product](https://patternflyelements.org/). 

```suggestion
We extended PatternFly 6 styles to Topology. Now, all PatternFly components are covered for the release, and all related design resources can be found in [our PatternFly 6 Figma kit.]()
```

**Review by mcoker — APPROVED** [review]
LGTM. Everything here looks good. I think if we dug in a little, there is a good amount more to add to component updates/features and bug fixes, but I don't know how granular we would want to be.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md (hunk 1 above)
Comment (edonehoo) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:17]: hm yeah, good point! I was trying to use a term that would be inclusive of components, but also layouts/patterns/charts/etc, but I wouldn't want to confuse things with another project. Components probably works well enough here to convey that

**Comment by edonehoo** [comment]
@mcoker yeah, for sure! I was seeing a lot of small things that I could add, but then I started wondering if getting too granular strays too far from the "highlights" nature of the page 🤔 But, also, if we want to be sure to highlight those specific things, then that value probably outweighs being higher-level/brief. Curious what others think

**Review by tlabaj — COMMENTED** [review]
You probably want to mention the masthead changes.

**Review by tlabaj — COMMENTED** [review]
We should also mention the Text being renamed to content change.

**Comment by edonehoo** [comment]
@tlabaj wdyt about how I added that in? Or do you think they need their own sections with more detail?

**Review by andrew-ronaldson — APPROVED** [review]
Approved to the max

**Comment by tlabaj** [comment]
> @tlabaj wdyt about how I added that in? Or do you think they need their own sections with more detail?

Looks good!

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**